docs/02-Install.md

```diff
@@ -1,13 +1,312 @@
+SeriesDescription **OR** scan parameter values (TE, TR, slice thickness, etc). 
+By default, this table is populated with entries for t1, t2, fMRI and DTI, and 
+the columns defining expected scan parameters (e.g. `TE_Range`) are defined very 
+broadly.  
+The `Scan_type` column values are defined in the `mri_scan_type` table 
+(e.g. 44=t1); do not include hyphens, spaces or periods in your 
+`mri_scan_type.Scan_type` column values.
+
+5. **`Config`** table
+
+The `Config` table can also be accessed and customized from the LORIS front-end, 
+under the Admin menu, the `Configuration` module. Here are the configuration
+settings that impact directly or indirectly the pipeline:
+
+Under the section `Study`:
+ * `ImagingUploader Auto Launch`
+ 
+Under the section `Paths`
+ * `Imaging Data`
+ * `LORIS-MRI Code`
+ * `MRI-Upload Directory`
+ * `MINC Files`
+ * `Images`
+ 
+ Under the section `Imaging Modules`
+  * `Patient ID regex`: Used by the DICOM Archive module to 
+  * `Patient name regex`
+  * `Lego phantom regex`
+  * `Living phantom regex`
+  * `Imaging Browser Tabulated Scan Types`: Used by Imaging Browser to decide what modalities to display the QC status on
+     
+ Under the section `Imaging Pipeline`:
+ * `Loris-MRI Data Directory`: Typically `/data/$PROJECT/bin/mri/`
+ * `Study Name`: Prefix used for all filenames inserted into Imaging Browser and the `files` table
+ * `User to notify when executing the pipeline`: User email address used when notification is to be sent by the pipeline
+ * `Full path to get_dicom_info.pl script`: Typically `/data/$PROJECT/bin/mri/dicom-arhive/get_dicom_info.pl`
+ * `Horizontal pictures creation`
+ * `NIFTII file creation`: Project choice to enable for automated NIFTII file creation
+ * `dcm2mnc binary to use when converting`
+ * `Path to Tarchives`
+ * `Upload creation of candidates`: Creates candidates into LORIS if enabled
+ * `Project batch management used`
+ * `If site is used`
+ * `Number of volumes in native DTI acquisitions`: Used by the DTIPrep pipeline
+ * `Scan type of native T1 acquisition`: Used by the DTIPrep pipeline
+ * `Max number of DTI rejected directions for passing QC`: Used by the DTIPrep pipeline
+ * `NIAK Path`: Used by the DTIPrep pipeline
+ * `Secondary QCed dataset`: Used by the DTIPrep pipeline
+
+
+### 2.2.2 LORIS
+
+1. **Imaging Uploader**
+
+
+Projects can upload scans and launch the pipeline in a variety of options 
+detailed in [PipelineOptions](PipelineOptions.md). 
+Irrespective of the project's choice as to whether the imaging scan is to be 
+uploaded through the Imaging Uploader GUI or not, pipeline insertion progress 
+can be consulted through a live 'Log Viewer' panel.
+Some settings need to be configured properly (`php.ini` variables, 
+`MRI-Upload Directory` and `ImagingUploader Auto Launch`), and are documented in 
+the [LORIS repository: Imaging Uploader Specification](https://github.com/aces/Loris/blob/master/modules/imaging_uploader/README.md).
```

Comment:
these links point now to master and they are only on minor branch, but eventually when minor is released it should be fine.